client/project/elements/query-form/checkbox/index.js

```diff
@@ -0,0 +1,40 @@
+import {route} from 'preact-router'
```

Comment:
@adriaanwm This is almost identical to `styled-form/checkbox` maybe they should share a base?